src/js/control-bar/control-bar.js

```diff
@@ -67,6 +68,7 @@ ControlBar.prototype.options_ = {
     'descriptionsButton',
     'subtitlesButton',
     'captionsButton',
+    'tracksButton',
```

Comment:
I assume that the default Control Bar won't have `subtitlesButton`, `captionsButton` *and* `tracksButton` - it'll be either/or?